unit/README.md

```diff
@@ -0,0 +1,14 @@
+To run the unit tests, simply call `jbuilder runtest`.
```

Comment:
I suppose you mean `dune` here.

Reply:
I have changed this to use the `./run-unit-tests` script now in any case, since it ensures that the `-runner sequential` tag is included and is necessary to prevent tests from failing.